### PR TITLE
Massive crate cost rebalance and fixed multiple cargo money exploits.

### PR DESCRIPTION
**Pull request by Isy232**
## Beginning Notes

Before we even start, I want to mention, this PR pretty much changes all parts of the cargo economy. I do not play a ton of cargo so if you want to go through the file diffs and point out issues with the costs of certain crates I would be happy to discuss or change these numbers accordingly.

I do however know basic math and know that economy exploits are bad. If we get rid of the exploits, we can allow cargo to order far more interesting things so they may be of much more use to the rest of the station.

## The Exploit

In the cargo packs file, AT THE VERY START.

var/cost = 700 // Minimum cost, or infinite points are possible.

Yet someone clearly didn't listen and broke economy...

## About The Pull Request

Before we start, you should know that each crate can be sold back to CC for 500 credits and each manifest if correctly stamped for 200 credits. This means you should subtract 700 credits from the cost of each crate then do the math.

I changed the value of selling a crate to 200 so it makes more sense, but the stupidity of what was previously done still holds.

The value of crates is utter shit and completely unbalanced, let me give a few suprising examples of how bad it is.

1: The combat knife crate.
- The combat knife single crate costs 1200 credits. 1200 - 700 = 500 credits per knife.
- The combat knife bulk crate contains 3 knives and costs 3000 credits. (3000 - 700) / 3 = 767 credits per knife.
- Yes, buying knives in bulk costs more money than buying them alone.
**- Yes, this applies to nearly every bulk crate.** 
- The worst example I found of this is the security auto rifle ammo crate, where buying in bulk costs **767** credits per unit where the single pack costs a **ONLY 50 FUCKING CREDITS** per unit.
- To solve this I applied basic math concepts and made all crates costs balanced.

2: Multiple crates cost less than 700 credits. This means **you can profit by buying these crates and selling the crate and manifest back to CC.**
- **Buying single medkit crates cost 330 credits. Selling the crate and manifest back to CC gives you 700 credits. You profit 370 credits from buying these crates.**
- To solve this I just purged all crates that cost less than 700 credits from the game or increased their cost to 800 or 900.

3: This PR rebalances the cost of nearly every crate ingame.
- Multiple crates that were needlessly expensive were made much less expensive so you may actually consider buying them.
- Crates now sell for a much more reasonable 200 credits instead of a nearly ludicrous 500. 
- All crate costs have been lowered by a minimum of 300 credits to compensate.
- Cost decreases on most crates. Important antagonist beating crates such as mindshields remain untouched.
- Most crates are now somewhat less expensive.

4: This PR also adds a few more types of crates I personally thought were not controversial. Here is the list:
- Adds materials crates. (Plasma and bluespace crystal not included, these costs are balanced with their export price).
- Bulk high traction floor tiles crate.
- Bulk bruisekits, burnkits, oxylosskits, toxinkits and regular kits. Removed all single packs of these crates due to the previously mentioned exploit.
- Moved riot and regular armour and helmet crates to 1 crate that now comes with helmets and armour.
- Bulk IV crate instead of IV single pack. The bulk is so cheap a single would be irrelevant.

Note: I did add more stack options for plasma and BS crystal. This is QOL for mappers and admins, these do not affect anything in normal gameplay.

## Why It's Good For The Game

This PR is meant to include all of the non controversial changes to crates and to fix the game. While it does add a few new ones, if you don't want a certain new crate in the game leave a comment on this PR and I'll atomize.

Now, on to the meat of it. The current list of crates is utter dogshit that does not follow basic math. This is a big improvement in every way.

I think this rebalancing, while may not be perfect, is a massive improvement to the state of the economy.

## Changelog
:cl:
del: Removed exploit that allowed you to get money by buying and selling crates.
tweak: Many single pack crates removed due to said exploit. Added bulk versions instead for very reasonable prices.
balance: Rebalanced a ton of crates according to math. Many crates had you paying more for the bulk option than the single pack.
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
Should probably nerf crate reselling in general while you're at it. Maybe 300 or 350?

**Comment by Isy232** [comment]
> Should probably nerf crate reselling in general while you're at it. Maybe 300 or 350?

Fuck, I gotta go back over like everything now. I shoulda thought of that sooner...

**Comment by Isy232** [comment]
The selling cost of a crate is now 200 points. Combined with the manifest you can get 400 points back by doing the crate right.

The cost of most crates has been lowered to compensate for the reduced amount of money back.

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1143,63 +1148,169 @@
...
+	contains = list(/obj/item/stack/sheet/mineral/diamond/five)
+	crate_name = "diamond sheets crate"
+
+/datum/supply_pack/materials/diamond20
+	name = "20 Diamond Sheets"
+	desc = "If you like high technology, these materials can help!"
+	cost = 11600
+	contains = list(/obj/item/stack/sheet/mineral/diamond/twenty)
+	crate_name = "bulk diamond sheets crate"
+
```
Comment (PowerfulBacon) [inline code/modules/cargo/packs.dm]: Stuff like this shouldn't be buyable and should remain the job of the miners

Comment (PowerfulBacon) [reply code/modules/cargo/packs.dm]: Titanium is eh but diamonds and uranium probably not so much, the station is in orbit around a planet rich in materials for a reason so nanotrasen shouldn't be shipping materials to them but the mining outpost going to nanotrasen

Comment (Isy232) [reply code/modules/cargo/packs.dm]: I think otherwise but I will atomize. All those packs have now been removed.

**Comment by Isy232** [comment]
Note: Removed minerals crates from this PR. I plan to atomize.

**Review by ike709 — DISMISSED** [review]

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -256,12 +254,26 @@
...
+	contains = list(/obj/item/clothing/suit/space,
+					/obj/item/clothing/head/helmet/space,
+					/obj/item/clothing/mask/breath)
+	crate_name = "space suit crate"
+	crate_type = /obj/structure/closet/crate/secure
+
+/datum/supply_pack/emergency/spacesuit/bulk
+	name = "Space Suit Crate"
+	desc = "Contains three aging suits from Space-Goodwill and a jetpack. Requires EVA access to open."
+	cost = 1900
```
Comment (ike709) [inline code/modules/cargo/packs.dm]: That's a pretty big discount considering 1 suit is 900

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -256,12 +254,26 @@
...
-	access = ACCESS_EVA
+	cost = 900
+	contains = list(/obj/item/clothing/suit/space,
+					/obj/item/clothing/head/helmet/space,
+					/obj/item/clothing/mask/breath)
+	crate_name = "space suit crate"
+	crate_type = /obj/structure/closet/crate/secure
+
+/datum/supply_pack/emergency/spacesuit/bulk
+	name = "Space Suit Crate"
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	name = "Bulk Space Suit Crate"
```

File: code/modules/cargo/packs.dm [hunk 4]
```diff
@@ -256,12 +254,26 @@
...
 					/obj/item/clothing/head/helmet/space,
 					/obj/item/clothing/mask/breath,
-					/obj/item/tank/jetpack/carbondioxide)
+					/obj/item/clothing/suit/space,
+					/obj/item/clothing/head/helmet/space,
+					/obj/item/clothing/mask/breath,
+					/obj/item/clothing/suit/space,
+					/obj/item/clothing/head/helmet/space,
+					/obj/item/clothing/mask/breath)
 	crate_name = "space suit crate"
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	crate_name = "bulk space suit crate"
```

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -300,36 +312,22 @@
...
-					/obj/item/ammo_box/c38/hotshot,
-					/obj/item/ammo_box/c38/iceblox)
-	crate_name = "ammo crate"
-
 /datum/supply_pack/security/armor
 	name = "Armor Crate"
-	desc = "Three vests of well-rounded, decently-protective armor. Requires Security access to open."
-	cost = 1000
+	desc = "Three vests of well-rounded, decently-protective armor and 3 brain buckets. Requires Security access to open."
+	cost = 1200
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1500
```

File: code/modules/cargo/packs.dm [hunk 6]
```diff
@@ -347,19 +345,10 @@
...
-	contains = list(/obj/item/clothing/head/helmet/sec,
-					/obj/item/clothing/head/helmet/sec,
-					/obj/item/clothing/head/helmet/sec)
-	crate_name = "helmet crate"
-
 /datum/supply_pack/security/laser
 	name = "Lasers Crate"
 	desc = "Contains three lethal, high-energy laser guns. Requires Security access to open."
-	cost = 2000
+	cost = 1700
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1800
```

File: code/modules/cargo/packs.dm [hunk 7]
```diff
@@ -622,26 +628,20 @@
 
 /datum/supply_pack/security/armory/riotarmor
 	name = "Riot Armor Crate"
-	desc = "Contains three sets of heavy body armor. Advanced padding protects against close-ranged weaponry, making melee attacks feel only half as potent to the user. Requires Armory access to open."
-	cost = 1500
+	desc = "Contains three sets of heavy body armor and helmets. Advanced padding protects against close-ranged weaponry, making melee attacks feel only half as potent to the user. Requires Armory access to open."
+	cost = 1800
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 2200
```

File: code/modules/cargo/packs.dm [hunk 8]
```diff
@@ -672,10 +672,11 @@
 	for(var/i in 1 to 10)
 		var/item = pick(contains)
 		new item(C)
+
 /datum/supply_pack/security/armory/smartmine
 	name = "Smart Mine Crate"
 	desc = "Contains three nonlethal pressure activated stun mines capable of ignoring mindshieled personnel. Requires Armory access to open."
-	cost = 4000
+	cost = 2000
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 2500
```

File: code/modules/cargo/packs.dm [hunk 9]
```diff
@@ -710,34 +712,28 @@
 /datum/supply_pack/security/armory/wt550_single
 	name = "WT-550 Auto Rifle Single-Pack"
 	desc = "Contains one high-powered, semiautomatic rifles chambered in 4.6x30mm. Requires Armory access to open."
-	cost = 2000
+	cost = 1400
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1600
```

File: code/modules/cargo/packs.dm [hunk 10]
```diff
@@ -710,34 +712,28 @@
...
 	contains = list(/obj/item/gun/ballistic/automatic/wt550)
 	small_item = TRUE
 
 /datum/supply_pack/security/armory/wt550
 	name = "WT-550 Auto Rifle Crate"
 	desc = "Contains two high-powered, semiautomatic rifles chambered in 4.6x30mm. Requires Armory access to open."
-	cost = 3500
+	cost = 3400
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 4000
```

File: code/modules/cargo/packs.dm [hunk 11]
```diff
@@ -801,27 +797,36 @@
...
-	desc = "The backbone of modern society. Barely ever ordered for actual engineering. Single Order."
-	cost = 800
-	small_item = TRUE
-	contains = list(/obj/item/clothing/gloves/color/yellow)
-
 /datum/supply_pack/engineering/powergamermitts
 	name = "Insulated Gloves Crate"
 	desc = "The backbone of modern society. Barely ever ordered for actual engineering. Contains three insulated gloves."
-	cost = 2000	//Made of pure-grade bullshittinium
+	cost = 1300	//Made of pure-grade bullshittinium
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1600	//Made of pure-grade bullshittinium
```

File: code/modules/cargo/packs.dm [hunk 12]
```diff
@@ -801,27 +797,36 @@
...
+	name = "Jetpack Crate"
+	desc = "For when you need to go fast in space!"
+	cost = 700
+	contains = list(/obj/item/tank/jetpack/carbondioxide)
+	crate_name = "jetpack crate"
+
+/datum/supply_pack/engineering/jetpack3
+	name = "Bulk Jetpack Crate"
+	desc = "Three jetpacks, enough for the whole gang!"
+	cost = 1300
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1800
```

File: code/modules/cargo/packs.dm [hunk 13]
```diff
@@ -1143,63 +1148,99 @@
...
 	desc = "Let some nice light in with fifty glass sheets!"
-	cost = 1000
+	cost = 800
 	contains = list(/obj/item/stack/sheet/glass/fifty)
 	crate_name = "glass sheets crate"
 
+/datum/supply_pack/materials/glass250
+	name = "250 Glass Sheets"
+	desc = "Holy SHEET thats a lot of glass!"
+	cost = 2400
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 3000
```

File: code/modules/cargo/packs.dm [hunk 14]
```diff
@@ -1143,63 +1148,99 @@
...
-	cost = 1000
+	cost = 800
 	contains = list(/obj/item/stack/sheet/iron/fifty)
 	crate_name = "iron sheets crate"
 
-/datum/supply_pack/materials/plasteel20
+/datum/supply_pack/materials/iron250
+	name = "250 Iron Sheets"
+	desc = "Enough Iron to rebuild half a station!"
+	cost = 2400
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 3000
```

File: code/modules/cargo/packs.dm [hunk 15]
```diff
@@ -1143,63 +1148,99 @@
...
-	cost = 7500
+	cost = 7200
 	contains = list(/obj/item/stack/sheet/plasteel/twenty)
 	crate_name = "plasteel sheets crate"
 
 /datum/supply_pack/materials/plasteel50
 	name = "50 Plasteel Sheets"
 	desc = "For when you REALLY have to reinforce something."
-	cost = 16500
+	cost = 17400
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 16000
```

File: code/modules/cargo/packs.dm [hunk 16]
```diff
@@ -1143,63 +1148,99 @@
...
 	contains = list(/obj/item/stack/sheet/plasteel/fifty)
 	crate_name = "plasteel sheets crate"
 
+/datum/supply_pack/materials/copper20
+	name = "20 Copper Sheets"
+	desc = "Makes your floors look nice and your circuitry run!"
+	cost = 800
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1000
```

File: code/modules/cargo/packs.dm [hunk 17]
```diff
@@ -1143,63 +1148,99 @@
...
-	cost = 2000
+	cost = 1700
 	contains = list(/obj/item/stack/sheet/mineral/wood/fifty)
 	crate_name = "wood planks crate"
 
 /datum/supply_pack/materials/bz
 	name = "BZ Canister Crate"
 	desc = "Contains a canister of BZ. Requires Toxins access to open."
-	cost = 8000
+	cost = 7000
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 8000
```

File: code/modules/cargo/packs.dm [hunk 18]
```diff
@@ -1208,7 +1249,7 @@
 /datum/supply_pack/materials/carbon_dio
 	name = "Carbon Dioxide Canister"
 	desc = "Contains a canister of Carbon Dioxide."
-	cost = 3000
+	cost = 800
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1200
```

File: code/modules/cargo/packs.dm [hunk 19]
```diff
@@ -1240,15 +1281,15 @@
...
 	contains = list(/obj/machinery/portable_atmospherics/canister/nitrogen)
 	crate_name = "nitrogen canister crate"
 	crate_type = /obj/structure/closet/crate/large
 
 /datum/supply_pack/materials/nitrous_oxide_canister
 	name = "Nitrous Oxide Canister"
 	desc = "Contains a canister of Nitrous Oxide. Requires Atmospherics access to open."
-	cost = 3000
+	cost = 1600
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 2400
```

File: code/modules/cargo/packs.dm [hunk 20]
```diff
@@ -1306,32 +1347,68 @@
...
 					/obj/item/reagent_containers/food/drinks/bottle/synthflesh,
 					/obj/item/reagent_containers/food/drinks/bottle/synthflesh)
 	crate_name = "rusty freezer"
 	crate_type = /obj/structure/closet/crate/freezer
 
-/datum/supply_pack/medical/firstaidbruises_single
+datum/supply_pack/medical/bruisekits
+	name = "Basic Treatment Kits Crate"
+	desc = "Contains three basic aid kits focused on basic types of damage in a simple way."
+	cost = 1000
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1400
```

File: code/modules/cargo/packs.dm [hunk 21]
```diff
@@ -1306,32 +1347,68 @@
...
+	small_item = TRUE
+	contains = list(/obj/item/storage/firstaid/regular,
+					/obj/item/storage/firstaid/regular,
+					/obj/item/storage/firstaid/regular)
+	crate_name = "basic wound treatment kits crate"
+
+datum/supply_pack/medical/bruisekits
+	name = "Bruise Treatment Kits Crate"
+	desc = "Contains three first aid kits focused on healing bruises and broken bones."
+	cost = 1000
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1400
```

File: code/modules/cargo/packs.dm [hunk 22]
```diff
@@ -1306,32 +1347,68 @@
...
+	small_item = TRUE
+	contains = list(/obj/item/storage/firstaid/brute,
+					/obj/item/storage/firstaid/brute,
+					/obj/item/storage/firstaid/brute)
+	crate_name = "brute treatment kits crate"
+
+/datum/supply_pack/medical/burnkits
 	name = "Bruise Treatment Kit Single-Pack"
```
Comment (ike709) [inline code/modules/cargo/packs.dm:1381]: Fix this

File: code/modules/cargo/packs.dm [hunk 23]
```diff
@@ -1306,32 +1347,68 @@
...
-	desc = "Contains one first aid kit focused on healing bruises and broken bones."
-	cost = 330
+	desc = "Contains three first aid kits focused on healing severe burns."
+	cost = 1000
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1400
```

File: code/modules/cargo/packs.dm [hunk 24]
```diff
@@ -1306,32 +1347,68 @@
...
+	crate_name = "burn treatment kits crate"
 
-/datum/supply_pack/medical/firstaidburns_single
-	name = "Burn Treatment Kit Single-Pack"
-	desc = "Contains one first aid kit focused on healing severe burns."
-	cost = 330
+/datum/supply_pack/medical/oxylosskits
+	name = "Oxygen Deprivation Kits Crate"
+	desc = "Contains three first aid kits focused on helping oxygen deprivation victims."
+	cost = 1000
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1400
```

File: code/modules/cargo/packs.dm [hunk 25]
```diff
@@ -1306,32 +1347,68 @@
...
-	contains = list(/obj/item/storage/firstaid/fire)
+	contains = list(/obj/item/storage/firstaid/o2,
+					/obj/item/storage/firstaid/o2,
+					/obj/item/storage/firstaid/o2)
+	crate_name = "oxygen deprivation treatment kits crate"
+
+/datum/supply_pack/medical/toxinkits
+	name = "Toxin Treatment Kits Crate"
+	desc = "Contains three first aid kits focused on healing damage dealt by heavy toxins."
+	cost = 1000
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1400
```

File: code/modules/cargo/packs.dm [hunk 26]
```diff
@@ -1306,32 +1347,68 @@
...
+	contains = list(/obj/item/storage/firstaid/toxin,
+					/obj/item/storage/firstaid/toxin,
+					/obj/item/storage/firstaid/toxin)
+	crate_name = "toxin treatment kits crate"
 
 /datum/supply_pack/medical/chemical
 	name = "Chemical Starter Kit Crate"
 	desc = "Contains thirteen different chemicals, for all the fun experiments you can make."
-	cost = 1700
+	cost = 600
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1000
```

File: code/modules/cargo/packs.dm [hunk 27]
```diff
@@ -1353,30 +1430,22 @@
 /datum/supply_pack/medical/defibs
 	name = "Defibrillator Crate"
 	desc = "Contains two defibrillators for bringing the recently deceased back to life."
-	cost = 2500
+	cost = 1200
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 1800
```

File: code/modules/cargo/packs.dm [hunk 28]
```diff
@@ -1353,30 +1430,22 @@
...
-	small_item = TRUE
-	contains = list(/obj/item/storage/firstaid/regular)
-
 /datum/supply_pack/medical/iv_drip
 	name = "IV Drip Crate"
-	desc = "Contains a single IV drip for administering blood to patients."
-	cost = 1000
+	desc = "Contains a three IV drips for administering blood to patients."
+	cost = 700
 	contains = list(/obj/machinery/iv_drip)
```
Comment (ike709) [inline code/modules/cargo/packs.dm]: Does not contain 3.

File: code/modules/cargo/packs.dm [hunk 29]
```diff
@@ -1353,30 +1430,22 @@
...
 	crate_name = "iv drip crate"
 
 /datum/supply_pack/medical/supplies
```
Comment (ike709) [inline code/modules/cargo/packs.dm:1449]: This is where my review ends. I'll finish it later.

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: 1 suit is 500, considering the value of the manifest its not actually a discount at all. (900 - 200 (manifest) - 200 (crate) = 500)

Well, the discount is in the time you save by not having to move 3 crates and approve 3 manifests.

File: code/modules/cargo/packs.dm (hunk 4 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Forgot that, oops

File: code/modules/cargo/packs.dm (hunk 3 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Forgot that, oops

File: code/modules/cargo/packs.dm (hunk 5 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Fair enough, changing it now.

File: code/modules/cargo/packs.dm (hunk 6 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Alright, looks fair

File: code/modules/cargo/packs.dm (hunk 8 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Aight

File: code/modules/cargo/packs.dm (hunk 7 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: I'll change it

File: code/modules/cargo/packs.dm (hunk 9 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Changing it

File: code/modules/cargo/packs.dm (hunk 10 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Alright

File: code/modules/cargo/packs.dm (hunk 11 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Sure

File: code/modules/cargo/packs.dm (hunk 17 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Sure

File: code/modules/cargo/packs.dm (hunk 18 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Done

File: code/modules/cargo/packs.dm (hunk 22 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm:1381]: You mean how the basic kits pack was supply_pack/medical/brutekits?
I saw that bug and fixed it unless there is another you mean. I don't see one here.

File: code/modules/cargo/packs.dm (hunk 28 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Oops, forgot to finish doing that one, fixed it up, my bad

**Comment by Isy232** [comment]
Made bulk crates a bit more economical now.

**Review by PowerfulBacon — DISMISSED** [review]
yolo

File: code/modules/cargo/packs.dm [hunk 30]
```diff
@@ -1353,29 +1432,25 @@
...
-	cost = 1000
-	contains = list(/obj/machinery/iv_drip)
+	desc = "Contains three IV drips for administering blood to patients."
+	cost = 800
+	contains = list(/obj/machinery/iv_drip,
+					/obj/machinery/iv_drip,
+					/obj/machinery/iv_drip)
 	crate_name = "iv drip crate"
 
+// IKE STOPPED HERE, REMOVE AFTER DONE
```
Comment (PowerfulBacon) [inline code/modules/cargo/packs.dm]:
```suggestion
```

File: code/modules/cargo/packs.dm [hunk 31]
```diff
@@ -1597,14 +1658,24 @@
...
+/datum/supply_pack/service/noslipfloor
+	name = "High-traction Floor Tiles"
+	desc = "Make an entire department not need to know the pain of slipping on a wet floor with 120 anti-slip floortiles!"
+	cost = 2000
+	contains = list(/obj/item/stack/tile/noslip/thirty,
+					/obj/item/stack/tile/noslip/thirty,
+					/obj/item/stack/tile/noslip/thirty,
+					/obj/item/stack/tile/noslip/thirty)
+	crate_name = "high-traction floor tiles crate"
+
```
Comment (PowerfulBacon) [inline code/modules/cargo/packs.dm]: These 2 have the same datum. Also floor tiles rather than floortiles

**Comment by Isy232** [comment]
Ah, I'm back, in a few hours I'll fix it

File: code/modules/cargo/packs.dm (hunk 30 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Removed this

File: code/modules/cargo/packs.dm (hunk 31 above)
Comment (Isy232) [reply code/modules/cargo/packs.dm]: Done

**Comment by Isy232** [comment]
I think its fixed up now

**Pull request merged**